### PR TITLE
Add Cassandra Meetup Organizer Handbook and Events Approval Checklist pages

### PR DESCRIPTION
**Pull request by nonstopdtop**
patch by Diogenese Topper; reviewed by -- for CASSANDRA-18899

### CODE REVIEW

File: site-content/source/modules/ROOT/pages/events/organizer-handbook.adoc [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+=== Organising Cassandra Events in an External Group
+
+If you are an organiser for another meetup group that isn’t exclusively about Cassandra but you are facilitating Cassandra-focused topics, then we would still love to promote your events to the Cassandra community. You can submit event details via https://docs.google.com/forms/d/e/1FAIpQLSeKY6FG27OlcF2jsUxs0-JyxtWWpYOTZnq7-78ZCTzgw-sVgg/viewform[this form^], and we will add them to the central events calendar.
+
+=== Building an Organising Team
+
+A diverse organising team is important for building a diverse and inclusive community; it also helps distribute the work of organising amongst more people. Remember the Prime Directive - Cassandra events are organised for the benefit of the community and not a single person or organisation.
+
+You can find additional organisers for your team in the Planet Cassandra Discord, by posting on social media, or by asking people you know who are interested in Apache Cassandra.
```
Comment (michaelsembwever) [inline site-content/source/modules/ROOT/pages/events/organizer-handbook.adoc]:
```suggestion
You can find additional organisers for your team in the ASF #cassandra, #cassandra-events and #cassandra-dev slack channels, in Planet Cassandra Discord, by posting on social media, or by asking people you know who are interested in Apache Cassandra.
```

File: site-content/source/modules/ROOT/pages/events/organizer-handbook.adoc [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+* The venue should be able to accommodate the number of expected attendees with seating for everyone, as well as include any features that are required for the format of the event (e.g. desks/tables for workshops that require attendees to use their laptops)
+* Avoid venues that could be potentially problematic for some attendees - this could include political or religious venues.
+
+=== Running Online Events
+
+Organisers may choose to run online events. The same guidelines apply to online events as to in-person ones. When selecting a platform to use for online events, ensure you select a platform that has adequate accessibility features for all attendees, as well as any features that the event format requires.
+
+=== Producing Swag
+
+You may produce an event or community-related swag for your community members at your discretion. While there are no restrictions on this, here are some guidelines to help organisers make good choices:
```
Comment (michaelsembwever) [inline site-content/source/modules/ROOT/pages/events/organizer-handbook.adoc]: this isn't correct.

swag does (should) be checked with the PMC and ASF Branding to ensure the ASF Trademarks are used correctly.

(the same also applies to all event materials, e.g. social graphics, flyers, banners, etc)

**Comment by nonstopdtop** [comment]
Thanks for review and suggestions @michaelsembwever. Suggestions acknowledged and changes have been made.

**Review by michaelsembwever — APPROVED** [review]

**Comment by michaelsembwever** [comment]
Committed as https://github.com/apache/cassandra-website/commit/c22fdf621b7e1ea7e5c6b2d76e7de5d1e8cb023b

**Pull request closed without merging**